recipes-containers/libnvidia-container-tools/libnvidia-container-tools_1.0.6.bb

```diff
@@ -0,0 +1,83 @@
+SUMMARY = "NVIDIA container runtime library"
+
+DESCRIPTION = "NVIDIA container runtime library \
+The nvidia-container library provides an interface to configure GNU/Linux \
+containers leveraging NVIDIA hardware. The implementation relies on several \
+kernel subsystems and is designed to be agnostic of the container runtime. \
+"
+
+HOMEPAGE = "https://github.com/NVIDIA/libnvidia-container"
+
+DEPENDS = " \
+    coreutils-native \
+    pkgconfig-native \
+    libcap \
+    elfutils \
+    libtirpc \
+    libseccomp \
+    ldconfig-native \
+"
+LICENSE = "GPLv3 & Apache-2.0"
+
+LIC_FILES_CHKSUM = "\
+    file://LICENSE;md5=3b83ef96387f14655fc854ddc3c6bd57 \
+    file://COPYING;md5=1ebbd3e34237af26da5dc08a4e440464 \
+    file://COPYING.LESSER;md5=3000208d539ec061b899bce1d9ce9404 \
+"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}-${PV}:"
+
+PR = "r1"
+
+NVIDIA_MODPROBE_VERSION = "396.51"
+ELF_TOOLCHAIN_VERSION = "0.7.1"
+LIBTIRPC_VERSION = "1.1.4"
+
+SRC_URI = " \
+    git://github.com/NVIDIA/libnvidia-container.git;name=libnvidia \
+    https://github.com/NVIDIA/nvidia-modprobe/archive/${NVIDIA_MODPROBE_VERSION}.tar.gz;name=modprobe \
+    https://sourceforge.net/projects/elftoolchain/files/Sources/elftoolchain-${ELF_TOOLCHAIN_VERSION}/elftoolchain-${ELF_TOOLCHAIN_VERSION}.tar.bz2;name=elftoolchain \
+    https://downloads.sourceforge.net/project/libtirpc/libtirpc/${LIBTIRPC_VERSION}/libtirpc-${LIBTIRPC_VERSION}.tar.bz2;name=libtirpc \
+    file://0001-Makefile-Fix-RCP-flags-and-change-prefix.patch \
+"
```

Comment:
Any chance that you can just make this recipe depend on `libtirpc` instead of pulling the sources in for it here?   Likewise for elftoolchain, although you'd have to write a recipe for that.  It's just nicer from a license compliance perspective.

Reply:
Its actually not using these sources I will remove them from here

Reply:
Check right now, we can still go further and remove nvidia modprobe from the recipe.

Logs:
NOTE: Executing Tasks
NOTE: Setscene tasks completed
WARNING: libnvidia-container-tools-1.0.6-r1 do_package_qa: QA Issue:      git://github.com/NVIDIA/libnvidia-container.git;name=libnvidia     https://github.com/NVIDIA/nvidia-modprobe/archive/396.51.tar.gz;name=modprobe     file://0001-Makefile-Fix-RCP-flags-and-change-prefix.patch : SRC_URI uses unstable GitHub archives [src-uri-bad]
NOTE: Tasks Summary: Attempted 905 tasks of which 891 didn't need to be rerun and all succeeded.

Summary: There was 1 WARNING message shown.